app/src/main/java/org/wikipedia/descriptions/DescriptionEditReviewView.kt

```diff
@@ -40,7 +39,7 @@ class DescriptionEditReviewView @JvmOverloads constructor(
         articleSubtitle!!.text = StringUtils.capitalize(description)
         articleExtract!!.text = StringUtil.fromHtml(pageSummary.extractHtml)
 
-        if (TextUtils.isEmpty(pageSummary.thumbnailUrl)) {
+        if (pageSummary.thumbnailUrl.isNullOrEmpty()) {
```

Comment:
For the thumbnails URLs, I think it would be better if we use `isNullOrBlank()` here.